doc/UsersGuide/source/CodeOverview.rst

```diff
@@ -127,7 +127,7 @@ The umbrella repository for the UFS WM is named ``ufs-weather-model``. Under thi
     ├── doc                                         -------- User Guide files
     ├── driver
     ├── fire_behavior                               -------- Community Fire Behavior Model
-    ├── FV3                                         -------- UFSAtm atmosphere model
+    ├── ufsatm                                      -------- UFSATM atmosphere model
```

Comment:
```suggestion
    ├── UFSATM                                      -------- UFSATM atmosphere model
```
Could you move this down so that everything remains in alphabetical order?

Reply:
There's also a `ci` directory, which should probably be added.